synopsys-task/task.json

```diff
@@ -13,199 +13,279 @@
+      "groupName": "optionalSettings"
     },
     {
       "name": "synopsys_bridge_path",
       "type": "string",
-      "label": "Synopsys Bridge Path",
+      "label": "Bridge Installation Path",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "A path where synopsys bridge will be configured."
+      "helpMarkDown": "A path where synopsys bridge will be configured.",
+      "groupName": "optionalSettings"
     },
     {
-      "name": "bridge_download_version",
+      "name": "include_diagnostics",
+      "type": "boolean",
+      "label": "Include Diagnostics",
+      "defaultValue": false,
+      "required": false,
+      "helpMarkDown": "All diagnostics files will be available to download when 'true' passed.",
+      "groupName": "optionalSettings"
+    },
+    {
+      "name": "scanType",
+      "type": "pickList",
+      "label": "Security Testing Solution: ",
+      "required": true,
+      "helpMarkDown": "Select required security testing solution",
+      "defaultValue": "",
+      "options": {
+        "": "",
+        "polaris": "Polaris",
+        "blackduck": "Black Duck",
+        "coverity": "Coverity Platform"
+      }
+    },
+    {
+      "name": "bridge_polaris_serverUrl",
       "type": "string",
-      "label": "Synopsys Bridge version",
+      "label": "Polaris URL",
       "defaultValue": "",
-      "required": false,
-      "helpMarkDown": "Bridge version to be downloaded."
+      "required": true,
+      "helpMarkDown": "URL for Polaris Server.",
+      "groupName": "polaris"
     },
     {
       "name": "bridge_polaris_accessToken",
       "type": "string",
       "label": "Polaris Access Token",
       "defaultValue": "",
-      "required": false,
-      "helpMarkDown": "Access token for Polaris."
+      "required": true,
+      "helpMarkDown": "Access token for Polaris.",
+      "groupName": "polaris"
     },
     {
       "name": "bridge_polaris_application_name",
       "type": "string",
-      "label": "Polaris Application Name",
+      "label": "Polaris Application",
       "defaultValue": "",
-      "required": false,
-      "helpMarkDown": "Application name in Polaris."
+      "required": true,
+      "helpMarkDown": "Application name in Polaris.",
+      "groupName": "polaris"
     },
     {
       "name": "bridge_polaris_project_name",
       "type": "string",
-      "label": "Polaris Project Name",
+      "label": "Polaris Project",
       "defaultValue": "",
-      "required": false,
-      "helpMarkDown": "Project name in Polaris."
+      "required": true,
+      "helpMarkDown": "Project name in Polaris.",
+      "groupName": "polaris"
     },
     {
       "name": "bridge_polaris_assessment_types",
       "type": "string",
-      "label": "Polaris Assessment Types",
+      "label": "Assessment Type",
       "defaultValue": "",
-      "required": false,
-      "helpMarkDown": "Polaris Assessment Types seperated by commas."
```

Comment:
WE need to add the href link. Please check mockup for hover text and JIRA for the href link